community-fund/COMMUNITY_FUND_POLICY.md

```diff
@@ -41,10 +41,11 @@ _All amounts are in United States dollars._
 Valid travel expenses are:
 
 * Airfare, train, or ferry tickets in economy/coach class only
-* Accommodations (Dates of the event plus one night prior are eligible for reimbursement)
+* Accommodations (dates of the event plus one night prior are eligible for reimbursement)
+* Visa
```

Comment:
```suggestion
* Visa fees (even in case the visa is denied)
```

Reply:
Big +1, we should clarify this in the doc